src/Bridges/Constraint/geomean.jl

```diff
@@ -150,24 +177,30 @@ function MOI.get(::MOI.ModelLike, ::MOI.ConstraintSet, bridge::GeoMeanBridge)
 end
 function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintFunction,
                  bridge::GeoMeanBridge{T, F, G, H}) where {T, F, G, H}
-    tub = MOI.get(model, attr, bridge.tubc)
-    rhs = MOI.constant(MOI.get(model, MOI.ConstraintSet(), bridge.tubc))
-    tub = MOIU.operate(-, T, tub, rhs)
-    t = MOIU.remove_variable(tub, bridge.xij[1])
-    f_scalars = Vector{MOIU.scalar_type(H)}(undef, bridge.d)
-    f_scalars[1] = t
     d = bridge.d
-    n = d - 1
-    l = ilog2(n)
-    num_lvars = 1 << (l - 1)
-    offset = num_lvars - 1
-    for j in 1:num_lvars
-        if 2j <= bridge.d
-            func = MOI.get(model, attr, bridge.socrc[offset + j])
-            func_scalars = MOIU.eachscalar(func)
-            f_scalars[2j] = func_scalars[1]
-            if 2j + 1 <= bridge.d
-                f_scalars[2j + 1] = func_scalars[2]
+    f_scalars = Vector{MOIU.scalar_type(H)}(undef, bridge.d)
+    tub = MOI.get(model, attr, bridge.tubc)
+    if d == 2
+        x = MOI.get(model, attr, bridge.nonneg)
+        f_scalars[2] = MOIU.eachscalar(x)[1]
+        f_scalars[1] = MOIU.operate(+, T, tub, f_scalars[2])
```

Comment:
If there was a constant then it was moved to the LessThan set and you need to add it back here